visual-inspection/inspection-frontend/src/components/ExplainInspection.vue

```diff
@@ -76,6 +76,7 @@ export default {
   },
   data() {
     return {
+      showExplainButton: process.env.VUE_APP_SHOW_EXPLAIN_BUTTON,
```

Comment:
```suggestion
      showExplainButton: JSON.parse(process.env.VUE_APP_SHOW_EXPLAIN_BUTTON),
```

Dann wird es gleich in einen Boolean konvertiert. (Ich habe einfach "javascript string to bool" gegoogelt und bin hier gelandet: https://stackoverflow.com/questions/263965/how-can-i-convert-a-string-to-boolean-in-javascript)